src/Api/Api.php

```diff
@@ -690,7 +690,14 @@ public function upload(Closure $callback, $single = false): array
         ];
 
         if (empty($files) === true) {
-            throw new Exception(t('upload.error.noFiles'));
+            $postMaxSize       = 1024 * 1024 * (int)ini_get('post_max_size');
```

Comment:
PHP allows shortcuts for byte values, including K (kilo), M (mega) and G (giga).

Do we need to convert this when ini settings `G` or especially `K`?

Reply:
You are completely right! I just created a new `Str::toBytes` method to handle that.